spring-cloud-starter-task-composedtaskrunner/src/test/java/org/springframework/cloud/task/app/composedtaskrunner/properties/ComposedTaskPropertiesTests.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.task.app.composedtaskrunner.properties;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * @author Glenn Renfro
+ */
+public class ComposedTaskPropertiesTests {
+
+	@Test
+	public void testGettersAndSetters() throws URISyntaxException{
+		ComposedTaskProperties properties = new ComposedTaskProperties();
+		properties.setComposedTaskProperties("aaa");
+		properties.setComposedTaskArguments("bbb");
+		properties.setIntervalTimeBetweenChecks(12345);
+		properties.setMaxWaitTime(6789);
+		properties.setDataFlowUri(new URI("http://test"));
+		properties.setGraph("ddd");
+		assertEquals("aaa", properties.getComposedTaskProperties());
+		assertEquals("bbb", properties.getComposedTaskArguments());
+		assertEquals(12345, properties.getIntervalTimeBetweenChecks());
+		assertEquals(6789, properties.getMaxWaitTime());
+		assertEquals("http://test", properties.getDataFlowUri().toString());
+		assertEquals("ddd", properties.getGraph());
+
+	}
+
+	@Test
+	public void testThreadDefaults() {
+		ComposedTaskProperties properties = new ComposedTaskProperties();
+		assertEquals(1, properties.getSplitThreadCorePoolSize());
+		assertEquals(60, properties.getSplitThreadKeepAliveSeconds());
+		assertEquals(Integer.MAX_VALUE, properties.getSplitThreadMaxPoolSize());
+		assertEquals(Integer.MAX_VALUE, properties.getSplitThreadQueueCapacity());
```

Comment:
Can we make these defaults constants so that we don't have to update them in multiple places if we change them?